theme/cerveza/src/components/_Snackbar.scss

```diff
@@ -0,0 +1,54 @@
+@use '../utilities';
+
+
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700');
+
+
+$snackColors: primary, secondary, success, warning, danger, info, dark, light;
+
+
+.brew-Snackbar {
+
+    &__content {
+        @include utilities.height-4();
+        border-radius: utilities.$border-radius;
+        box-sizing: border-box;
+        display: flex;
+        align-items: center;
+        font-family: Roboto, sans-serif;
+        min-width: utilities.size(normal) * 21;
+        height: utilities.size(normal) * 3;
+        padding: utilities.size(normal);
+    }
+
+    &__icon {
+        font-size: utilities.size(xx-large);
+        margin-right: utilities.size(normal);
+    }
+
+    &__message {
+        padding: 0;
+        font-size: utilities.size(small);
```

Comment:
Using the `Typography` component for the message would prevent you from having to do things like set the font family and font size in your css.

Reply:
Noted, will fix.